composer.json

```diff
@@ -16,15 +16,15 @@
         "php": ">=7.3",
         "phpcompatibility/php-compatibility": "^9.1",
         "squizlabs/php_codesniffer": "^3.6",
-        "phpro/grumphp": "^1.5",
         "drupal/coder": "^8.3",
         "cweagans/composer-patches": "^1.7",
         "composer/xdebug-handler": "^3.0",
-        "phpmd/phpmd": "^2.10"
+        "phpmd/phpmd": "^2.10",
```

Comment:
Is `phpmd/phpmd` still needed or it could go down with GrumPHP? I see it's required by both, Toolkit & QA Auto. We're not using it. We simply run `phpcs`